### ISSUE TITLE
change Domain

### ISSUE THREAD

**Issue by happyreacer** [nextcloud/all-in-one#322]
Hi,
is it possible to change my Domain? Or what is an alternative way? Possibly via a backup in a new setup?

**Comment by szaimen** [comment]
Hi, this is currently not possible and I am also not sure if it makes sense for the project as you are not able to change the domain in the android app and desktop client either, afaik. Either way I'll leave it open for now but it will not be implemented in the near future.

**Comment by michaelso** [comment]
how about a change of internal ip? i did change my server's IP in the local network after setting up nextcoud-aio, and not sure how to change the local internal ip for accessing the aio panel

**Comment by szaimen** [comment]
> how about a change of internal ip? i did change my server's IP in the local network after setting up nextcoud-aio, and not sure how to change the local internal ip for accessing the aio panel

are you talking about the automatic login button that still has the old ip-address?

**Comment by michaelso** [comment]
no, it is in the https://domain.name/settings/admin/overview
[Open Nextcloud AIO Interface ↗](https://IP:8080/api/auth/getlogin?token=xxx)
Hope there is a way to update the IP in this case when the server is moved internally

**Comment by szaimen** [comment]
> no, it is in the https://domain.name/settings/admin/overview
> [Open Nextcloud AIO Interface ↗](https://IP:8080/api/auth/getlogin?token=xxx)

Yes, I thought so. See this how to change the value there: https://github.com/nextcloud/all-in-one/issues/130#issuecomment-1012105239
